packages/financial-templates-lib/src/logger/Formatters.ts

```diff
@@ -14,12 +14,19 @@ export function errorStackTracerFormatter(logEntry: LogEntry) {
 // make it more readable. If something goes wrong in parsing the object (it's too large or something else) then simply
 // return the original log entry without modifying it.
 export function bigNumberFormatter(logEntry: LogEntry) {
+  type SymbolRecord = Record<string | symbol, any>;
   try {
-    iterativelyReplaceBigNumbers(logEntry);
+    // Out is the original object if and only if one or more BigNumbers were replaced.
+    const out = iterativelyReplaceBigNumbers(logEntry);
+
+    // Because winston depends on some non-enumerable symbol properties, we explicitly copy those over, as they are not
+    // handled in iterativelyReplaceBigNumbers. This only needs to happen if logEntry is being replaced.
+    if (out !== logEntry)
+      Object.getOwnPropertySymbols(logEntry).map((symbol) => (out[symbol] = (logEntry as SymbolRecord)[symbol]));
```

Comment:
I wanted to call this out -- there are some non-enumerable, `Symbol` props that winston depends on (like `Symbol("level")`), so those must be copied over if the object is swapped out.